forte/data/data_store.py

```diff
@@ -1183,8 +1182,58 @@ def get_length(self, type_name: str) -> int:
             delete_count = self.__deletion_count.get(type_name, 0)
             return len(self.__elements[type_name]) - delete_count
 
+    def get_bisect_range(
+        self, search_list: SortedList, range_span: Tuple[int]
+    ) -> Optional[List]:
+        """
+        Perform binary search on the specified list for target entry class.
```

Comment:
I think the docstring is not very clear about the usage of the function. Also, we can make the function private if we only use it internally.